code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -303,11 +303,6 @@
 	synd_mob.update_icons()
 	return 1
 
-/datum/game_mode/nuclear/check_win()
-	if(nukes_left == 0)
-		return 1
-	return ..()
-
 
```

Comment:
/datum/game_mode/nuclear/check_win() was removed because im also trying to cut down on check_win. And this doesnt actually do anything.